Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -236,57 +114,68 @@ bool RunProcess(const FString& ExecutablePath, const FString& Arguments)
 	return true;
 }
 
-bool SpatialGDKGenerateInteropCode(const ClassHeaderMap& InteropGeneratedClasses)
+FString GenerateIntermediateDirectory()
 {
-	if (!CheckClassNameListValidity(InteropGeneratedClasses))
+	const FString CombinedIntermediatePath = FPaths::Combine(*FPaths::GetPath(FPaths::GetProjectFilePath()), TEXT("Intermediate/Improbable/"), *FGuid::NewGuid().ToString(), TEXT("/"));
+	FString AbsoluteCombinedIntermediatePath = FPaths::ConvertRelativePathToFull(CombinedIntermediatePath);
+	FPlatformFileManager::Get().GetPlatformFile().CreateDirectoryTree(*AbsoluteCombinedIntermediatePath);
+
+	return AbsoluteCombinedIntermediatePath;
+}
+
+bool SpatialGDKGenerateInteropCode()
+{
+	const USpatialGDKEditorToolbarSettings* SpatialGDKToolbarSettings = GetDefault<USpatialGDKEditorToolbarSettings>();
```

Comment:
Will this ever be null pointer? How would this function be ran without the toolbar?